src/Components/FilterNavbar/FilterNavbar.jsx

```diff
@@ -19,6 +25,18 @@ export function FilterNavbar() {
                         <a href="#">
                             <span className="title">Price</span>
                         </a>
+                        <div className="nav-sublists">
+                            <ul>
+                                <li>
+                                    <input type="radio" name="c1" id="c1" checked = {initialState.sortBy && initialState.sortBy === "LOW_TO_HIGH"} onClick={() => filterAction({ type : "SORT_PRICE", payload : "LOW_TO_HIGH"})}/>
```

Comment:
Really liking the structure, one thing you can do is dispatch strings constants. Not necessary, just makes it easy to scale. Keep up!